pkg/printers/internalversion/printers_test.go

```diff
@@ -235,6 +235,8 @@ func testPrinter(t *testing.T, printer printers.ResourcePrinter, unmarshalFunc f
 		TypeMeta:   metav1.TypeMeta{APIVersion: "v1", Kind: "Pod"},
 		ObjectMeta: metav1.ObjectMeta{Name: "foo"},
 	}
+	// our decoder defaults, so we should default our expected object as well
+	legacyscheme.Scheme.Default(obj)
```

Comment:
independent?

Reply:
fixing the bug in versioning.codec that was skipping defaulting made this test fail… the expected object was missing defaults that were present in the decoded object